_data/issues/input-mask.yml

```diff
@@ -0,0 +1,9 @@
+title: Input mask
+alert_type: info
+items:
+- date: 2023-09-29
+  summary: Users had trouble correctly typing dates into the input field.
+  summary_additional:
+  affectsScreenReaders: true
+  issue_repo: uswds
+  issue_number: 5525
```

Comment:
@jaclinec 
I changed the issue number on this from https://github.com/uswds/uswds/pull/5227 to https://github.com/uswds/uswds/pull/5525 here. I did this because [#5227](https://github.com/uswds/uswds/pull/5227) is an outdated PR that we plan on splitting into multiple issues and PRs. Issue https://github.com/uswds/uswds/pull/5525 felt like the better option because it was a summary of the component findings. 

Related, I am wondering if we should break this single item into multiple "known issues" to match the granularity of our other known issues. We already have some of the listed concerns entered as [issues in the repo](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Package%3A+input+mask%22+). Maybe we can link them up to the [main findings issue](https://github.com/uswds/uswds/pull/5525), add some detail, and put them as "known issues" items here? Curious what you think!

Reply:
@amyleadem I added detail and renamed [#5481 ](https://github.com/uswds/uswds/issues/5481)to match the template we use for 'known issues.' I renamed it "Input mask: Recovering from an error is difficult due to lack of feedback."

Reply:
Added issue #5481 and updated the issue summary in [99b1c55](https://github.com/uswds/uswds-site/pull/2402/commits/99b1c5565cad43d309a7eb3b42b7ca3790d57990)